src/gossip_agent/mod.rs

```diff
@@ -108,6 +108,7 @@ impl GossipAgent {
         waku_host: Option<String>,
         waku_port: Option<String>,
         waku_addr: Option<String>,
+        boot_node_id: Option<&str>,
```

Comment:
They are `boot_node_addr`, and would you be able to update this to a Vec instead of a single address?

Reply:
oh right, the id is just the last part